modules/compute_instance/versions.tf

```
@@ -15,8 +15,11 @@
  */
 
 terraform {
-  required_version = ">=0.12.7, <0.14"
+  required_version = ">=0.13.0"
   required_providers {
-    google = ">= 2.7, <4.0"
+    google = ">= 3.43, <4.0"
+  }
+  provider_meta "google" {
+    module_name = "blueprints/terraform/terraform-google-vm-compute_instance/v6.0.0"
```

Sorry I realized we probably want clearer separation between modules and submodules.

```suggestion
    module_name = "blueprints/terraform/terraform-google-vm:compute_instance/v6.0.0"
```

---

good idea, done